### DISCUSSION TITLE
HTMLtoDOCX is not a function

### DISCUSSION THREAD

**Discussion by LSantos06** [privateOmega/html-to-docx#90] [Q&A] [answered]
Hi! I'm trying to use this library with React, but I keep getting this 'HTMLtoDOCX is not a function' error. I've already tried the following alternatives for importing the library:

`import * as HTMLtoDOCX from 'html-to-docx';`
`import HTMLtoDOCX from 'html-to-docx';`
`const HTMLtoDOCX = require('html-to-docx');`

How can I properly import the library in my project using npm? Thanks!

**Comment by privateOmega** [comment]
@LSantos06 Which version of html-to-docx are you trying with? Also could you please post additional details like React Version, Loader you are using etc?

**Reply by LSantos06** [reply, marked as answer]
@privateOmega Actually I've successfully imported the library with `const HTMLtoDOCX = require('../dist/html-to-docx.umd');`, just like it's illustrated in the examples folder.
